kubernetes/deployment.yml

```diff
@@ -0,0 +1,16 @@
+apiVersion: extensions/v1beta1
```

Comment:
deployment is not an extension anymore, it is now part of the core k8s api. See https://kubernetes.io/docs/concepts/workloads/controllers/deployment/

Reply:
WOW. It did work for me, but going to change that